### PR TITLE
Noon Theme added to Develop (New Feature)

### PR DESCRIPTION
**Pull request by cameronblaikie**
## Description

 
Added a high contrast theme (NOON) 
## How To Test This

Select the Noon theme and then test all functionality i.e drop downs, text boxes, etc. 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Mac-Rae** [comment]
@cameronblaikie Unable to load theme

**Comment by Mac-Rae** [comment]
Appears it's not included within the Robo build:theme command ether so unable to rebuild manually

File: themes/SuiteP/css/Dawn/color-palette.scss [hunk 1]
```diff
@@ -7,9 +7,9 @@ $color-3: #333333;
 $color-4: #337AB7;
 $color-5: #3C763D;
 $color-6: #444444;
-$color-7: #534D64;
+$color-7: #196579;
 $color-8: #555555;
-$color-9: #5D5670;
+$color-9: #48808F;
```
Comment (Mac-Rae) [inline themes/SuiteP/css/Dawn/color-palette.scss]: Looks like Dawn was swapped out for SuiteASSURED colours?

**Comment by Mac-Rae** [comment]
Few issues I've found. Let me know if you need further clarification other than the screenshot

cant see x on meeting menu
cant see x on meeting menu

Menu not visable
anothernovoisable
notvisable

Gray on Gray
nothighcontrast

Buttons dont have same hover colour
savebutton2
savebutton1

**Comment by Mac-Rae** [comment]
2 More issues (Others above have been resolved)

Screenshot from 2020-01-22 09-40-06
Screenshot from 2020-01-22 09-39-43

**Comment by horus68** [comment]
What is the objective on level compliance for color contrast
On tests and tools https://github.com/salesagility/SuiteCRM/issues/4776

**Pull request closed without merging**